### PR TITLE
feat(cli): add reports field to statoscope config

### PR DESCRIPTION
**Pull request by amalitsky**
**WIP**

Add ability to add reports to *generate* and *serve* CLI commands

Relates to #156

### CODE REVIEW

File: packages/cli/src/utils.ts [hunk 1]
```diff
@@ -9,6 +9,9 @@ import {
 import { parseChunked } from '@discoveryjs/json-ext';
 import normalizeCompilation from '@statoscope/webpack-model/dist/normalizeCompilation';
 import { Webpack } from '@statoscope/webpack-model/webpack';
+import { Report } from '@statoscope/types/types/custom-report';
+import { Config } from '@statoscope/types/types/config';
+import { mergeReports } from './commands/injectReport';
```
Comment (smelukov) [inline packages/cli/src/utils.ts:14]: it's better to move `mergeReports` into utils ;)

**Comment by amalitsky** [comment]
Probably test are failing due to the absolute file paths used in `report.html`. Need to think what to do about those

File: packages/types/types/config/index.d.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { Config as ValidatorConfig } from '../validation/config';
+import { Report } from '../custom-report';
 
 export type Config = {
   silent?: boolean;
   validate?: ValidatorConfig;
+  reports?: Report<unknown, unknown>[];
```
Comment (smelukov) [inline packages/types/types/config/index.d.ts:7]: Maybe `generate.reports`?

Comment (amalitsky) [reply packages/types/types/config/index.d.ts]: mb `generate.customReports`? So that it matches command line arguments

**Comment by smelukov** [comment]
@amalitsky if statoscope.config has an error (e.g. invalid syntax) then there is no error
can you fix this please?

**Comment by smelukov** [comment]
@amalitsky thanks!

**Pull request merged**